src/Client.php

```diff
@@ -485,8 +496,8 @@ public function upsertAccount(array $account): CallResult
         $payload = [
             "identification" => $this->accountIdentifiersToArray(
                 new AccountIdentified(
-                    isset($account["accountId"]) ? (string) $account["accountId"] :  null,
-                    isset($account["domain"]) ? (string) $account["domain"] : null
+                    isset($account["accountId"]) ? (string)$account["accountId"] : null,
+                    isset($account["domain"]) ? (string)$account["domain"] : null
```

Comment:
Did you manage to fix the styling with `vendor/bin/phpcbf --standard=PSR12 src`?

Reply:
![image](https://user-images.githubusercontent.com/36775261/123771707-9124c980-d8cb-11eb-9be5-cdc9b3832faf.png)

:+1: